packages/ui/src/theme/css/component/autocomplete.scss

```diff
@@ -0,0 +1,82 @@
+%menu-space-shared {
+  padding-block-end: var(
+    --amplify-components-autocomplete-menu-space-shared-padding-block-end
+  );
+  padding-block-start: var(
+    --amplify-components-autocomplete-menu-space-shared-padding-block-start
+  );
+  padding-inline-end: var(
+    --amplify-components-autocomplete-menu-space-shared-padding-inline-end
+  );
+  padding-inline-start: var(
+    --amplify-components-autocomplete-menu-space-shared-padding-inline-start
+  );
+}
+
+.amplify-autocomplete {
+  position: relative;
+}
+
+.amplify-autocomplete__menu {
+  position: absolute;
+  z-index: 99;
+
+  width: var(--amplify-components-autocomplete-menu-width);
+  margin-top: var(--amplify-components-autocomplete-menu-margin-top);
+  background-color: var(
+    --amplify-components-autocomplete-menu-background-color
+  );
+  border-color: var(--amplify-components-autocomplete-menu-border-color);
+  border-width: var(--amplify-components-autocomplete-menu-border-width);
+  border-style: var(--amplify-components-autocomplete-menu-border-style);
+  border-radius: var(--amplify-components-autocomplete-menu-border-radius);
+
+  &__header,
+  &__footer,
+  &--empty {
+    @extend %menu-space-shared;
+  }
+
+  &--loading {
+    @extend %menu-space-shared;
+
+    align-items: var(
+      --amplify-components-autocomplete-menu-loading-align-items
+    );
+    gap: var(--amplify-components-autocomplete-menu-loading-gap);
```

Comment:
Interesting that loading needs these styles, but empty, footer, and header do not. Do we need them?

Reply:
We do not need them for the others. Loading contains a Loader, so need some extra css work to align it with the text
![Screen Shot 2022-10-27 at 3 06 21 PM](https://user-images.githubusercontent.com/40295569/198407106-e6002197-9352-4bb3-83df-59e9a698afbd.png)